### ISSUE TITLE
List not (complete) visible, if field focused

### ISSUE THREAD

**Issue by nubu** [TheSmiths-Widgets/ts.chat#2]
Hey,

clicking in the field resulting in moving it's above the keyboard-  right behavier :) 

BUT

In this case i cannot scroll the list with the entrys already available until to it's top. Any idea about it?

Best, Nico

**Comment by KtorZ** [comment]
Hey!

Does it happen on iOS or Android or both ?

**Comment by nubu** [comment]
Hi,

on ios. I've tried to use a second scrollview instead of a tableview (don't see that i need the tableview specific functions here) but that doesn't work. Seems the scrolling of the tableview internally is handled different from the scrollview.

**Comment by KtorZ** [comment]
Indeed, there is a special manual workaround on iOS with the scrollView.
Also, I will have to switch from the tableView to a listView, it will probably be nicer.

About your problem, I promise to have a look during the weekend. If you have any other info, do not hesitate :)

**Comment by nubu** [comment]
Hey,

thanks for that! :)

I think (or thought) that scrollview will be the best for this - cause we don't really need the functionallity of either the tableview or the listview - do we? 

I think conversations are always just "bubbles" with text in it, so i don't see any reason for using a TeableviewRow or listelement feature. But maybe i'm wrong?
(Don't know if the performance of a scrollview containing just other views elements (layout:vertical) in it is better than rows in a tableview or elements of a listview.)

Best, nico
